.github/workflows/auto-release-rc.yml

```
@@ -0,0 +1,95 @@
+# Compares the version in pyproject.toml to tags on the repo. If the tag doesn't exist, a new tag is created, which
+# then triggers the normal "on tag" release automation in the build job
+name: Auto Tag RC
+
+on:
+  push:
+    branches:
+      - develop
+      - tna-in-releases2
+
+concurrency:
+  group: rc-release-check
+
+jobs:
+  check-version:
+    name: Check version increment
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clean workspace
+        uses: Chia-Network/actions/clean-workspace@main
+
+      - name: Checkout current branch
+        uses: actions/checkout@v4
+        with:
+          # Need PACKAGE_ADMIN_PAT token so when the tag is created, the tag automation runs
+          token: ${{ secrets.PACKAGE_ADMIN_PAT }}
```

```suggestion
          # Need REPO_COMMIT token so when the tag is created, the tag automation runs
          token: ${{ secrets. REPO_COMMIT }}
```
I think we created this "Commit" token since the other workflows were made, but I think it should still be able to trigger the same workflows, while being a bit lower perms/scope